nuxeo-features/nuxeo-automation/nuxeo-automation-jsf/src/main/java/org/nuxeo/ecm/automation/jsf/operations/CancelCreateDocumentInUI.java

```diff
@@ -33,7 +33,8 @@
  */
 @Operation(id = CancelCreateDocumentInUI.ID, category = Constants.CAT_UI, requires = Constants.SEAM_CONTEXT, label = "Cancel Document Creation in UI", description = "Cancels a document creation in UI, "
         + "as if user was hitting the 'Cancel' button on a the document creation form. "
-        + "It returns the parent document, that would have been the created document container.", aliases = { "Seam.CancelCreateDocumentInUI" })
+        + "It returns the parent document, that would have been the created document container.", aliases = {
+                "Seam.CancelCreateDocumentInUI" })
```

Comment:
Please do cleanups like reformatting in separate commits, so that the main commit can focus on the actual changes. This is for easier review.